hack/ci/run-conformance-tests.sh

```diff
@@ -62,7 +62,9 @@ elif [[ $provider == "openstack" ]]; then
   EXTRA_ARGS="-openstack-domain=${OS_DOMAIN}
     -openstack-tenant=${OS_TENANT_NAME}
     -openstack-username=${OS_USERNAME}
-    -openstack-password=${OS_PASSWORD}"
+    -openstack-password=${OS_PASSWORD}
+    -openstack-application-credential-id=${OS_APPLICATION_CREDENTIAL_ID}
```

Comment:
This requires a change in your setup. Did you had the change to add this to your env?